simple_app.py

```diff
@@ -205,7 +205,7 @@ def mainloop():
                        selectcolor="lightblue"
                        )
     for i in range(1):
```

Comment:
Need to remove this whole loop, look it's still instantiating a bot with fake stats